samples/adminconfig/infrastructure.json

```diff
@@ -1,45 +1,65 @@
 {
+    "metrics": [
+        {
+            "name": "cost",
+            "type": "numeric",
+            "units": "US Dollar per TB per month",
+            "scale": 
+            {
+                "min": 0,
+                "max": 500
+            }
+        },
+        {
+            "name": "distance",
+            "type": "numeric",
+            "units": "km",
+            "scale":
+            {
+                "min": 0,
+                "max": 20000
+            }
+        }
+    ],
     "infrastructure": [
         {
             "attribute": "storage-cost",
             "description": "neverland object store",
+            "metricsName": "cost",
             "value": "100",
-            "type": "numeric",
-            "units": "US Dollar per TB per month",
             "object": "fybrikstorageaccount",
             "instance": "account-neverland"
         },
         {
             "attribute": "storage-cost",
             "description": "theshire object store",
+            "metricsName": "cost",
             "value": "90",
-            "type": "numeric",
-            "units": "US Dollar per TB per month",
             "object": "fybrikstorageaccount",
             "instance": "account-theshire"
         },
         {
-            "attribute": "bandwidth",
-            "description": "bandwidth between neverland and theshire",
-            "value": "5",
-            "type": "numeric",
-            "units": "Mbps",
+            "attribute": "distance",
+            "description": "distance between neverland and theshire",
+            "metricsName": "distance",
+            "value": "5000",
+            "argObjects": ["fybrikstorageaccount","fybrikstorageaccount"],
```

Comment:
This attribute refers to regions, so why do we need the argObjects and why are they set to `fybrikstorageaccount`?
Will we also have values for storage-account->cluster and cluster->cluster ?

Reply:
Right, reverting this change.